tests/v04/test_omero.py

```diff
@@ -0,0 +1,28 @@
+import json
+from pathlib import Path
+
+from ome_zarr_models.v04.omero import Channel, Omero, Window
+
+
+def test_load_example_json():
+    with open(Path(__file__).parent / "data" / "omero_example.json") as f:
```

Comment:
we should put these JSON documents in our tests module

Reply:
Isn't this the test module? What filepath within the repo are you rpoosing instead?

Reply:
quite right, `Path(__file__).parent / "data" / "omero_example.json"` will resolve to a path inside the tests module, but we could also store the JSON inline as python strings, and then they can be imported without requiring tests to do IO

Reply:
(another option would be keep the JSON files as JSON files, but make our tests access them via a pytest fixture that does the IO once per test session)

Reply:
The advantage of storing in a .json file instead of an inline string is getting nice linting/syntax highlighting for it. At the moment this is only read in once per session - perhaps we can leave as is for now, and if/when we add more similar tests think about how to de-duplicate the machinery?

Reply:
+1 to storing JSON as JSON :) and this is not a blocker, we can handle this when we apply a good first-pass normalization of all our tests